code/modules/atmospherics/gasmixtures/reactions.dm

```diff
@@ -786,3 +797,389 @@ nobiliumsuppression = INFINITY
+	name = "Delta gas formation"
+	id = "delta_gas_formation"
+
+/datum/gas_reaction/delta_gas_formation/init_reqs()
+	min_requirements = list(
+		/datum/gas/bz = MINIMUM_MOLE_COUNT,
+		/datum/gas/tritium = MINIMUM_MOLE_COUNT,
+		"TEMP" = 150
+	)
+
+/datum/gas_reaction/delta_gas_formation/react(datum/gas_mixture/air, datum/holder)
+	var/list/cached_gases = air.gases
+	var/temperature = air.temperature
+	var/old_heat_capacity = air.heat_capacity()
+	var/heat_efficency = min(temperature * 0.01, cached_gases[/datum/gas/tritium][MOLES], cached_gases[/datum/gas/bz][MOLES])
+	var/energy_used = heat_efficency * 100
+	ASSERT_GAS(/datum/gas/delta_gas,air)
+	if ((cached_gases[/datum/gas/tritium][MOLES] - heat_efficency < 0 ) || (cached_gases[/datum/gas/bz][MOLES] - heat_efficency < 0)) //Shouldn't produce gas from nothing.
+		return NO_REACTION
+	if(air.temperature >= 250)
+		return NO_REACTION
+	cached_gases[/datum/gas/tritium][MOLES] -= heat_efficency * 5
+	cached_gases[/datum/gas/bz][MOLES] -= heat_efficency * 0.25
+	cached_gases[/datum/gas/delta_gas][MOLES] += heat_efficency * 2
+
+	if(energy_used > 0)
+		var/new_heat_capacity = air.heat_capacity()
+		if(new_heat_capacity > MINIMUM_HEAT_CAPACITY)
+			air.temperature = max(((temperature * old_heat_capacity + energy_used) / new_heat_capacity),TCMB)
+		return REACTING
+
+/datum/gas_reaction/gamma_gas_formation
+	priority = 16
+	name = "Gamma gas formation"
+	id = "gamma_gas_formation"
+
+/datum/gas_reaction/gamma_gas_formation/init_reqs()
+	min_requirements = list(
+		/datum/gas/bz = MINIMUM_MOLE_COUNT,
+		/datum/gas/hydrogen = MINIMUM_MOLE_COUNT,
+		"TEMP" = 150
+	)
+
+/datum/gas_reaction/gamma_gas_formation/react(datum/gas_mixture/air, datum/holder)
+	var/list/cached_gases = air.gases
+	var/temperature = air.temperature
+	var/old_heat_capacity = air.heat_capacity()
+	var/heat_efficency = min(temperature * 0.01, cached_gases[/datum/gas/tritium][MOLES], cached_gases[/datum/gas/bz][MOLES])
+	var/energy_used = heat_efficency * 100
+	ASSERT_GAS(/datum/gas/gamma_gas,air)
+	if ((cached_gases[/datum/gas/hydrogen][MOLES] - heat_efficency < 0 ) || (cached_gases[/datum/gas/bz][MOLES] - heat_efficency < 0)) //Shouldn't produce gas from nothing.
+		return NO_REACTION
+	if(air.temperature >= 250)
+		return NO_REACTION
+	cached_gases[/datum/gas/hydrogen][MOLES] -= heat_efficency * 5
+	cached_gases[/datum/gas/bz][MOLES] -= heat_efficency * 0.25
+	cached_gases[/datum/gas/delta_gas][MOLES] += heat_efficency * 2
+
+	if(energy_used > 0)
+		var/new_heat_capacity = air.heat_capacity()
+		if(new_heat_capacity > MINIMUM_HEAT_CAPACITY)
+			air.temperature = max(((temperature * old_heat_capacity + energy_used) / new_heat_capacity),TCMB)
+		return REACTING
+
+/datum/gas_reaction/gas_x_formation
+	priority = 12
+	name = "Gas X formation"
+	id = "gas_x_formation"
+
+/datum/gas_reaction/gas_x_formation/init_reqs()
+	min_requirements = list(
+		/datum/gas/bz = MINIMUM_MOLE_COUNT,
+		/datum/gas/freon = MINIMUM_MOLE_COUNT,
+		"TEMP" = 10
```

Comment:
Is this intended as a minimum or a maximum?

Reply:
is a min

Reply:
aight that'll work, 10k seems a bit low for a minimum, is this to account for freon/bz formation?

Reply:
kinda yes, is there also to not put if(temp < 10 && temp > 50) return NO_REACTION
but i guess i could remove that, maybe make it a bit higher too

Reply:
If you want support for min/max temps in reactions I can fenangle it if you want

Reply:
no need don't worry